www/source/blog/2018-03-21-0550-Release.html.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Habitat 0.55.0 Released
+date: 2018-03-21
+author: Elliott Davis
+tags: release notes
+category: product
+classes: body-article
+---
+
+# Habitat 0.55.0 Release Notes
```

Comment:
I think you can just remove this line; it's rendering YUGE because `h1`, and it looks like it's been omitted in exactly 50% of previous release-oriented blog posts. But since it's clear from the title what this is about, it's probably fine just to leave it out.